internal/http.go

```diff
@@ -6,7 +6,38 @@
 
 package internal // import "go.mongodb.org/mongo-driver/internal"
 
-import "net/http"
+import (
+	"net"
+	"net/http"
+	"time"
+)
 
 // DefaultHTTPClient is the default HTTP client used across the driver.
-var DefaultHTTPClient = &http.Client{}
+var DefaultHTTPClient = &http.Client{
+	Transport: &http.Transport{
```

Comment:
Any reason not to just use `http.DefaultTransport`?

Reply:
removed ForceAttemptHTTP2